storage/src/main/java/tech/pegasys/teku/storage/server/kvstore/KvStoreConfiguration.java

```diff
@@ -51,7 +51,7 @@ public class KvStoreConfiguration {
 
   public static final int DEFAULT_MAX_BACKGROUND_JOBS = 6;
   public static final int DEFAULT_BACKGROUND_THREAD_COUNT = 6;
-  public static final long DEFAULT_CACHE_CAPACITY = 8 << 20; // 8MB
+  public static final long DEFAULT_CACHE_CAPACITY = 134217728; // 128MB
```

Comment:
i left this in `DEFAULT_CACHE_CAPACITY = 128 << 20; // 128MB` in the other PR, would be more clear

Reply:
why do the bit shifting when we can actually specify a constant? I'm not strongly in favour of any tbh, but I'd rather use a constant number and have the comment clarifying that.

Reply:
its a little clearer than 134xxxxxx == 128mb

Reply:
alright, changed back to the old notation